pkg/utils/httpserver/server_test.go

```diff
@@ -26,7 +27,8 @@ func TestStartServer(t *testing.T) {
 			logger.Error("failed to write response", zap.Error(err))
 		}
 	}))
-	go StartServer(ctx, logger, "test", "8999", m)
+	mgr := manager.New()
+	go StartServer(ctx, logger, mgr, "test", "8999", m)
```

Comment:
add to mgr instead of spawning go routine as done in main services.

Reply:
done

Reply:
done